Bellerophon/Bellerophon/Sources/BPConfig.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  BellerophonConfig.swift
+//  Bellerophon
+//
+//  Created by Satinder Singh on 11/17/17.
+//
+
+import UIKit
+
+/// Configuration for Bellerophon
```

Comment:
This comment is pretty useless. It's obvious from the struct name that it's a config for Bellerophon. I'd perhaps use this comment to describe why this config exists, where it's used and how it's used.

Reply:
updated